File: src/cs/postgres_serial.cs
using System;
using System.Globalization;
using Npgsql;

namespace Calc
{
  class Program
  {
    static void Main(string[] Args)
    {
      NpgsqlConnection conn;
      NpgsqlCommand cmd;
      int i;
      double x, prob;
      CultureInfo.DefaultThreadCurrentCulture = new CultureInfo("en-US");
      try
      {        
        conn = new NpgsqlConnection("Host=localhost;Username=root;Password=root123;Database=gaussian");
        conn.Open();
        cmd = conn.CreateCommand();
        cmd.CommandText = "DROP TABLE IF EXISTS cs_serial; CREATE TABLE cs_serial(id SERIAL " +
        "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL)";
        cmd.ExecuteNonQuery();
        for (i = -500; i <= 500; i++)
        {
          x = (double)i / 100.0;
          prob = Gaussian.GaussianCDF(0, 1, x);
          cmd.CommandText = $"INSERT INTO cs_serial(z_score, cumulative_distribution) VALUES ({x:0.00}, {prob:0.000000});";
          cmd.ExecuteNonQuery();
        }
        conn.Close();
      }
      catch (System.Exception e)
      {
        Console.WriteLine(e.Message);
      }
    }
  }
}

File: src/js/postgres_serial.js
const { gaussianCDF } = require("./gaussian"),
  pg = require("pg");

async function main() {
  let pool, conn;
  try {
    pool = new pg.Pool({
      host: "localhost",
      database: "gaussian",
      user: "root",
      password: "root123",
    });
    conn = await pool.connect();
    conn.query(
      "DROP TABLE IF EXISTS js_serial; CREATE TABLE js_serial(id SERIAL " +
        "PRIMARY KEY, z_score REAL NOT NULL, cumulative_distribution REAL NOT NULL);"
    );
    for (let i = -500; i <= 500; i++) {
      let x = i / 100,
        prob = gaussianCDF(0, 1, x);
      conn.query(
        `INSERT INTO js_serial(z_score, cumulative_distribution) VALUES (${x}, ${prob.toFixed(
          6
        )});`
      );
    }
    conn.query("SELECT version", () => conn.end());
  } catch (err) {
    console.log(err.message);
  }
}

main();
